tests/test_inbox.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+import pytest
+from flask import url_for
+from flask.testing import FlaskClient
+
+from hushline.db import db
+from hushline.model import Message, User, Username
+
+# Define test passwords
+TEST_USER_PASSWORD = os.getenv("TEST_USER_PASSWORD", "TestPassword123!")
+OTHER_USER_PASSWORD = os.getenv("OTHER_USER_PASSWORD", "OtherUserPass456!")
+
+
+@pytest.fixture()
+def setup_user_data(user: User) -> None:
+    """Fixture to create primary and alias usernames and a message for the authenticated user."""
+    primary_username = Username(
+        user_id=user.id,
+        _username="primary_user",
+        is_primary=True,
+        show_in_directory=True,
+    )
+    alias_username = Username(
+        user_id=user.id,
+        _username="primary_user_alias",
+        is_primary=False,
+        show_in_directory=True,
+    )
+    db.session.add_all([primary_username, alias_username])
+    db.session.flush()
+
+    message = Message(
+        username_id=primary_username.id,
+        content="Test message for deletion.",
+    )
+    db.session.add(message)
+    db.session.commit()
+
+    return primary_username, alias_username, message
+
+
+@pytest.fixture()
+def other_user() -> User:
```

Comment:
This can be added to `conftest` as `user_2` which would establish a repeatable naming pattern

Reply:
done

Reply:
Did you drop a commit? They're still in the test file

Reply:
This is outdated.